SimPEG/Utils/modelutils.py

```diff
@@ -145,3 +147,44 @@ def surface2ind_topo(mesh, topo, gridLoc='CC', method='nearest', fill_value=np.n
             raise NotImplementedError('surface2ind_topo not implemented for Quadtree or 1D mesh')
 
     return mkvc(actind)
+
+
+def surfaceLayerIndex(mesh, topo, index=0):
```

Comment:
Would `surface_layer_index` work with you? Trying to shift to more consistent pep8 naming

Reply:
yes

Reply:
would you mind updating the name? I think this is the last one :) Thanks @fourndo